Can't start Sipi following instructions in README

If `SIPI_VERSION` is not set, I get an error message, but it's not clear what I should put in `SIPI_VERSION`. I tried `develop`, but that didn't work, either:

```
% cd git/knora-api/sipi
% make up           
docker-compose up -d sipi
WARNING: The SIPI_VERSION variable is not set. Defaulting to a blank string.
ERROR: no such image: dhlabbasel/sipi:: invalid reference format
make: *** [up] Error 1
% export SIPI_VERSION=develop
% make up                    
docker-compose up -d sipi
Pulling sipi (dhlabbasel/sipi:2.0.1)...
ERROR: manifest for dhlabbasel/sipi:2.0.1 not found: manifest unknown: manifest unknown
make: *** [up] Error 1
```

--- a/sipi/Makefile
+++ b/sipi/Makefile
@@ -6,8 +6,8 @@ print-env-file: env-file ## prints the env file used by sipi
 
 .PHONY: env-file
 env-file: ## write the env file used by sipi
-	@echo SIPI_VERSION=$(SIPI_VERSION) > .env
-	@echo DOCKERHOST=$(DOCKERHOST) > .env
+	@echo SIPI_VERSION=v$(SIPI_VERSION) > .env
+	@echo DOCKERHOST=$(DOCKERHOST) >> .env
 
 .PHONY: up
 up: env-file ## start sipi

